homework/english/semenenko_artur/lesson1/grammar.md

```diff
@@ -0,0 +1,89 @@
+### Homework_1
+
+
+
+## Раскройте скобки, употребляя глаголы в Present Continuous или в Present Simple.
+
+
+
+Is your mum working today?
+
+Why are these boys talking now?
+
+Where does he live?
+
+Does their mother cook very well?
+
+Lili is not rich, she does not drive a Mercedes.
+
+Let´s go out. It is raining  now.
+
+I have to go now. It is getting  dark.
+
+A young man usually stands  at the window.
+
+Ann usually makes doll dresses herself.
+
+ Is his wife having  coffee at the moment?
```

Comment:
i thought "at the moment" is dont meaning "now"
so it seems to need simple prasent form

Reply:
but shes still enjoying it)

Reply:
VolhaZh, i'm sure it is not a mistake. "to be" + V1. And she having coffee right now.

Reply:
ok